doc(pe): Document policy options (#601)
armory/docs: content/en/docs/plugin-guide/plugin-policy-engine/packages/spinnaker.http.authz/tasks/type.undoRolloutManifest.md

---
title: "Task Type: undoRolloutManifest"
linktitle: "undoRolloutManifest"

> description: "Policy controls whether or not a rollback that is triggered from outside a spinnaker pipeline (e.g. from the 'Clusters' tab of an application) can run."

Reviewer: ```suggestion
description: "Policy that controls whether or not a rollback that is triggered from outside of a pipeline (such as from the **Clusters** tab of an application) can run."
```
